packages/cli/src/Server.ts

```diff
@@ -2204,7 +2205,7 @@ async function getExecutionsCount(countFilter: IDataObject): Promise<{ count: nu
 
 async function generateInstanceId(){
```

Comment:
if you set the return type to `string` here, would you still need the `as string` on line 234

Reply:
as it is an async function it must return a `Promise<string | undefined>`, therefore the `as string` stay needed to force the stringtype